### DISCUSSION TITLE
show cursor position in document coordinates - ENHANCEMENT

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#3536] [Ideas]
Add keyboard shortcut to show cursor position relative to origin BOTTOM LEFT (for example "SHIFT-m" ) in addition to already  implemented shortcut "m" with origin TOP LEFT.

**Comment by GitHubRulesOK** [comment]
Seems reasonable  but really only of value to PDF editors 
Related to enhancements of measure tool see https://github.com/sumatrapdfreader/sumatrapdf/issues/750

**Reply by ghost** [reply]
I agree. the problem arrrised during -add-text or -crop using Coherent PDF command line utilites. CPDF often uses bottom left as origin.
in these cases transformation of the coordinates has to be done first. it would make life easier.
